config/locales/en.yml

```diff
@@ -89,6 +89,15 @@ en:
           msg: "There's no meeting currently running in this room"
           title: "Meeting not running"
     bigbluebutton: BigBlueButton
+    meetings:
+      delete:
+        success: "The meeting was successfully deleted."
```

Comment:
new message: "The register was successfully deleted!"

Reply:
done